build/deploy/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM busybox
+FROM buildpack-deps:jessie-curl
```

Comment:
I see. If we deployed using a musl-dynamic image, I'd have a hard time
figuring out why we didn't just opt for a statically-linked musl build in
the first place.

On Sun, Dec 13, 2015 at 5:43 PM Ben Darnell <notifications@github.com>
wrote:

> In build/deploy/Dockerfile
> <https://github.com/cockroachdb/cockroach/pull/3427#discussion_r47451726>:
>
> > @@ -1,4 +1,4 @@
> > -FROM busybox
> > +FROM buildpack-deps:jessie-curl
>
> We need *a* libc.so; I'm not sure whether we specifically require glibc
> or if a dynamically-linked musl would work. There are also "alpine with
> glibc" images out there which might work, but seem kind of hacky.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/cockroachdb/cockroach/pull/3427/files#r47451726>.
>